inc/ticket.class.php

```diff
@@ -1886,7 +1886,7 @@ function manageValidationAdd($input) {
          if (count($validations_to_send)) {
             $values                = array();
             $values['tickets_id']  = $this->fields['id'];
-            if ($input['id'] != $this->fields['id']) {
+            if (isset($input['id']) && $input['id'] != $this->fields['id']) {
```

Comment:
I'm asking because the initial comportement has changed; I do not know what is the good solution, even if I agree with you :)